src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -89,6 +89,8 @@
   //Max size for stack checks
   private static final int MAX_STACKSIZE = 1024;
 
+  public static BlockCapsule blockCap = null;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static blockCap" field final ([squid:S1444](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1444))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1398&issues=AWWA_HwielKyZq-pP3d-&open=AWWA_HwielKyZq-pP3d-)

[ISSUE_KEY]: AWWA_HwielKyZq-pP3d-